docs/source/integrations/deltalake_versioning.md

```diff
@@ -0,0 +1,107 @@
+# Data versioning with Delta Lake
+
+[Delta Lake](https://delta.io/) is an open-source storage layer that brings reliability to data lakes by adding a transactional storage layer on top of the data stored in cloud storage. It allows for ACID transactions, data versioning, and rollback capabilities. Delta table is the default table format in Databricks and is typically used for data lakes, where data is ingested either incrementally or in batch.
+
+This tutorial explores how to use Delta tables in your Kedro workflow and how to leverage the data versioning capabilities of Delta Lake.
+
+## Prerequisites
+
+In this example, you will use the `spaceflights-pandas` starter project which has example pipelines and datasets to work with. If you haven't already, you can create a new Kedro project using the following command:
+
+```bash
+kedro new --starter spaceflights-pandas
+```
+
+Kedro offers various connectors in the `kedro-datasets` package to interact with Delta tables: [`pandas.DeltaTableDataset`](https://github.com/kedro-org/kedro-plugins/blob/main/kedro-datasets/kedro_datasets/pandas/deltatable_dataset.py), [`spark.DeltaTableDataset`](), [`spark.SparkDataset`](), [`databricks.ManagedTableDataset`](), and [`ibis.FileDataset`]()ß support the delta table format. In this tutorial, we will use the `pandas.DeltaTableDataset` connector to interact with Delta tables using Pandas DataFrames. To install `kedro-datasets` alongwith dependencies required for Delta Lake, add the following line to your `requirements.txt`:
+
+```bash
+kedro-datasets[pandas-deltatabledataset]
+```
+
+Now, you can install the project dependencies by running:
+
+```bash
+pip install -r requirements.txt
+```
+
+## Using Delta tables in catalog
+
+To use Delta tables in your Kedro project, you can update the `base/catalog.yml` to use `type: pandas.DeltaTableDataset` for the datasets you want to save as Delta tables. For this example, let us update the `model_input_table` dataset in the `base/catalog.yml` file:
+
+```yaml
+model_input_table:
+  type: pandas.DeltaTableDataset
+  filepath: data/02_intermediate/model_input_table
```

Comment:
Isn't the original path in `data/03_primary`?

Reply:
Yep, updated